src/_posts/2021-02-11-cass-changelog_4.markdown

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Apache Cassandra Changelog #4 | February 2021"
+date:   2020-02-11
+author: the Apache Cassandra Community
+categories: blog
+---
+
+Our monthly roundup of key activities and knowledge to keep the community informed.
+
+![Apache Cassandra Changelog Header](https://cassandra.apache.org/img/changelog_header.jpg)
+
+## **Release Notes**
```

Comment:
Remove `**` from both sides of heading. They are unnecessary.

Reply:
Removed all occurrences as requested 👍